app/views/links/new.html.slim

```diff
@@ -1,4 +1,4 @@
-- - model_class = Link
```

Comment:
Note: this redundant hyphen is generated by html2slim, so I removed it manually.